usr/src/cmd/zdb/zdb.c

```diff
@@ -1578,8 +1583,54 @@ dump_deadlist(dsl_deadlist_t *dl)
 static avl_tree_t idx_tree;
 static avl_tree_t domain_tree;
 static boolean_t fuid_table_loaded;
-static boolean_t sa_loaded;
-sa_attr_type_t *sa_attr_table;
+static objset_t *sa_os = NULL;
```

Comment:
I see, it is used only for assertions, but we couldn't make the identical assertions with sa_attr_table.  Personally, I don't think those assertions are super important, but I'll defer to you.